src/types/libdef/npm-custom/react-testing-library_v5.x.x.js

```diff
@@ -13,44 +14,47 @@ declare module 'react-testing-library' {
     collapseWhitespace?: boolean,
   };
 
-  declare type SelectorMatchOptions = {selector?: string} & TextMatchOptions;
+  declare type SelectorMatchOptions = { selector?: string } & TextMatchOptions;
 
   declare type AllByText = (
     text: TextMatch,
     options?: TextMatchOptions
   ) => Array<HTMLElement>;
 
   declare type QueryByText = (
-      text: TextMatch,
-      options?: TextMatchOptions,
-    ) => ?HTMLElement;
+    text: TextMatch,
+    options?: TextMatchOptions
+  ) => ?HTMLElement;
 
   declare type GetsAndQueries = {|
     getByTestId: (id: TextMatch, options?: TextMatchOptions) => HTMLElement,
-    getByText: (
-      text: TextMatch,
-      options?: SelectorMatchOptions,
-    ) => HTMLElement,
+    getByText: (text: TextMatch, options?: SelectorMatchOptions) => HTMLElement,
     getByPlaceholderText: (
       text: TextMatch,
-      options?: TextMatchOptions,
+      options?: TextMatchOptions
     ) => HTMLElement,
     getByLabelText: (
       text: TextMatch,
-      options?: SelectorMatchOptions,
+      options?: SelectorMatchOptions
+    ) => HTMLElement,
+    getByTitle: (
+      text: TextMatch,
+      options?: SelectorMatchOptions
     ) => HTMLElement,
```

Comment:
This is the function I wanted to add. The rest is prettier doing its magic and that I don't know how to disable :)